templates/javascript-ethers/scripts/read-beacon.js

```diff
@@ -55,6 +55,7 @@ async function main() {
   }
 
   try {
+    // TODO: More user friendly value format
```

Comment:
I like this idea and we could use something like `ethers.utils.formatEther(value)` to convert BigNumber to a more readable number.

Reply:
Good idea. Will do.